.changeset/swift-colts-drive.md

```diff
@@ -0,0 +1,10 @@
+---
+'@directus/system-data': patch
+'@directus/types': patch
+'docs': patch
```

Comment:
AFAIK we don't include the docs package in changesets because it isn't actually published on npm.

```suggestion
```